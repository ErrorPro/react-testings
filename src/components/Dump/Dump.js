import React from 'react';

export default props => (
  <button onClick={props.onClick}>{props.label}</button>
);
