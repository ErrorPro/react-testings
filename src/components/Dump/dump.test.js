import React from 'react';
import Dump from './Dump';
import renderer from 'react-test-renderer';

test('dump components renders initially', () => {
  const component = renderer.create(
    <Dump onChange={() => {}} label="Hi i am a dump component" />
  );

  expect(component.toJSON()).toMatchSnapshot();
});
