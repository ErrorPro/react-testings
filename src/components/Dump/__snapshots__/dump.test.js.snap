// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dump components renders initially 1`] = `
<button
  onClick={undefined}
>
  Hi i am a dump component
</button>
`;
